library/src/morpheus-master/js/src/sakai.morpheus.profile.popup.js

```diff
@@ -85,7 +85,7 @@ profile.ignoreFriendRequest = function (removerId, friendId) {
  */
 profile.attachPopups = function (jqArray) {
 
-    if (!(jqArray instanceof jQuery)) {
+    if (!(jqArray instanceof $PBJQ)) {
```

Comment:
Yes it is, but the initial check against jQuery was failing depending on what JS scope you were calling from. This was the most reliable comparison.